addons/pharma/stringtable.xml

```diff
@@ -3084,19 +2980,7 @@
             <Russian>Выпить Кофеин</Russian>
         </Key>
         <Key ID="STR_KAT_Pharma_EACA_DescShort">
-            <English>Encourages clot formation</English>
-            <Czech>Podporuje tvorbu sraženin</Czech>
-            <Italian>Favorisce la formazione di coaguli</Italian>
-            <French>Favorise la formation de caillots</French>
-            <Korean>혈전 형성을 촉진합니다.</Korean>
-            <German>Fördert die Gerinnungsbildung</German>
-            <Chinesesimp>促进凝块形成</Chinesesimp>
-            <Polish>Wspomaga powstawanie skrzepów</Polish>
-            <Japanese>血液凝固を促進し、出血を抑えます</Japanese>
-            <Spanish>Favorece la coagulación</Spanish>
-            <Portuguese>Favorece a coagulação</Portuguese>
-            <Finnish>Edistää hyytymien muodostumista</Finnish>
-            <Russian>Способствует образованию тромбов</Russian>
+            <English>Stitches banadaged and clotted wounds</English>
```

Comment:
```suggestion
            <English>Stitches banadaged wounds</English>
```
clotted wounds are basically bandaged, dont confuse people there is other "type of wound"